token-support-tokenx-validation/src/main/kotlin/no/nav/tms/token/support/tokenx/validation/tokendings/VerifierWrapper.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.tms.token.support.tokenx.validation.tokendings
+
+import com.auth0.jwk.Jwk
+import com.auth0.jwk.JwkProvider
+import com.auth0.jwt.JWT
+import com.auth0.jwt.algorithms.Algorithm
+import com.auth0.jwt.interfaces.DecodedJWT
+import com.auth0.jwt.interfaces.JWTVerifier
+import java.security.interfaces.RSAPublicKey
+
+internal class VerifierWrapper(
```

Comment:
Jeg blir litt forvirret av wrapper begrepet her, det forteller meg ikke så mye om hva denne klassen har ansvaret for. Det er jo litt egen logikk her, hva beskriver denne logikken best? Kunne et annet navn vært mer beskrivende, f.eks. TokenVerifier?

Reply:
Var en konsekvens av en del reformatering. Wrapper var et mer passende navn opprinnelig, men enig med at det ikke lenger er riktig.